mas-foundation/src/main/java/com/ca/mas/foundation/MAS.java

```diff
@@ -74,6 +96,7 @@ private static synchronized void init(@NonNull final Context context) {
         new MASConfiguration(appContext);
         ConfigurationManager.getInstance().setMobileSsoListener(new AuthenticationListener(appContext));
         registerMultiFactorAuthenticator(otpMultiFactorAuthenticator);
+        addLifeCycleListener(new JWKPreLoadListener());
```

Comment:
How to disable it if I don't want to preload it? It must be an option for the developer.